contour/how-to-guides/configuring-contour-with-loadbalancer-tls-termination.hbs.md

```diff
@@ -91,6 +86,8 @@ Follow these steps to configure your Tanzu Application Platform:
     contour:
```

Comment:
```suggestion
    contour:
      infrastructure_provider: aws
```

The key `contour.envoy.service.aws.LBType` only takes effect if `infrastructure_provider = aws`